packages/cli/src/rpc/explain/buildContext.ts

```diff
@@ -60,7 +60,7 @@ export default async function buildContext(searchResults: SearchRpc.SearchResult
 
       if (codeSnippets.has(event.location)) continue;
 
-      const snippets = await lookupSourceCode(event.location);
+      const snippets = await lookupSourceCode(result.directory, event.location);
```

Comment:
The directory argument here is crucial because it's how the source file path (which is a relative path) can be resolved. You may want to dig into `lookupSourceCode` and satisfy yourself that it's doing the right thing.